docs/configuration/config-modification.md

```diff
@@ -45,7 +46,10 @@ interface ConfigInterface {
 - **show?: Partial<ShowConfig>**
 
   This field contains configuration responsible for rendering specific parts of the AsyncAPI component.
-  All except the `sidebar` fields are set to `true` by default.
+  All except the `sidebar` and `messageExamples` fields are set to `true` by default.
```

Comment:
I think you meant `false`

Reply:
Yes, the default for `sidebar` and `messageExamples` is `false`. Note the "All except...".

But we could also change it to the following to make it clearer:

> The `sidebar` and `messageExamples` fields are set to `false` by default. The default for all other fields is `true`.

Reply:
yeah, that sounds better, thanks 🙏🏼

Reply:
Changed